Formula/mingw-w64.rb

```diff
@@ -19,6 +19,10 @@ class MingwW64 < Formula
   # Apple's makeinfo is old and has bugs
   depends_on "texinfo" => :build
 
+  option "with-posix", "Compile with posix thread model"
+  option "without-i686", "Compile without support for 32bit"
```

Comment:
`without-i686` and `without-x86_64` are out of the question. We want to provide a feature-full experience (no build from source) for as many users as possible.